tutor/templates/config.yml

```diff
@@ -21,6 +21,7 @@ RUN_REDIS: true
 RUN_SMTP: true
 CMS_HOST: "studio.{{ LMS_HOST }}"
 CONTACT_EMAIL: "contact@{{ LMS_HOST }}"
+CONTACT_MAILING_ADDRESS: "contact@{{ LMS_HOST }}"
```

Comment:
I'd rather avoid adding new Tutor config variables for every LMS/CMS setting. Instead, we should create good defaults that people can override with plugins. Could we default to `{{ PLATFORM_NAME }} - {{ CONTACT_EMAIL }}` for the mailing address?